### PR TITLE
Handle incomplete reads from StreamReader

### PR DESCRIPTION
**Pull request by livercat**
When reading a relatively large file from zk (~5 kb), StreamReader isn't guaranteed to completely read it in one go. Add chunked read with default timeout of 3 seconds.

### CODE REVIEW

**Comment by livercat** [comment]
I ended up having to leave DEFAULT_READ_TIMEOUT at 1 second, because at 3 seconds test_barrier fails. I wasn't able to run tests locally to properly debug the test.

File: aiozk/connection.py [hunk 1]
```diff
@@ -174,15 +174,18 @@ def handle_write(write_future):
...
         remaining_size = size
-        payload = b''
         end_time = time() + self.read_timeout
+        payload = []
         while remaining_size and (time() < end_time):
-            chunk = await self.reader.read(remaining_size)
-            payload += chunk
-            remaining_size -= len(chunk)
+            remaining_time = end_time - time()
+            done, _ = await asyncio.wait([self.reader.read(remaining_size)], timeout=remaining_time)
```
Comment (cybergrind) [inline aiozk/connection.py]: Oh, I've realized that here we have to close not finished futures here (assigned `_` in your code).
Or we can use `asyncio.wait_for` + handle `asyncio.TimeoutError`

**Comment by livercat** [comment]
@cybergrind does this PR need anything else? If possible, a new release with this fix would be great.

**Comment by cybergrind** [comment]
sure

**Pull request merged**